src/components/Article.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <div>
+    <h1>{title}</h1>
```

Comment:
Usually there should be only one `h1` element per page. So it's better to remove `h1` from `App` component.